packages/firestore/test/integration/api/type.test.ts

```diff
@@ -96,7 +96,7 @@ apiDescribe('Firestore', (persistence: boolean) => {
       const validateSnapshots = !persistence;
       await expectRoundtrip(
         db,
-        { a: 1, b: NaN, c: Infinity, d: persistence ? 0.0 : -0.0 },
+        { a: 1, b: NaN, c: Infinity, d: persistence ? 0.0 : 0.0 },
```

Comment:
was this intentional?

Reply:
No, I suspect this was performed by a tool, but I can't pinpoint the tool that would have changed this. Reverted.